Sources/Views/LoadingView.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  LoadingView.swift
+//  PovioKit
+//
+//  Created by Toni Kocjan on 04/11/2021.
+//  Copyright © 2021 Povio Labs. All rights reserved.
+//
+
+import UIKit
+
+/// Adapt any view into a loading view.
+open class LoadingView<V: UIView>: UIControl {
+  public let view: V
+  public let activityIndicator: UIActivityIndicatorView
+  let animations: Animations?
+  
+  public init(
+    adapt view: V,
+    activityIndicatorStyle style: UIActivityIndicatorView.Style = .white,
```

Comment:
We should use `.medium` here since .`white` is going to be deprecated.

Reply:
Good point 👌